src/webviews/apps/shared/components/onboarding/onboarding-item.ts

```diff
@@ -0,0 +1,107 @@
+import { css, html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { when } from 'lit/directives/when.js';
+
+@customElement('gl-onboarding-item')
+export class GlOnboardingItem extends LitElement {
+	static override readonly styles = css`
+		:host {
+			display: flex;
+			align-items: center;
+			font-size: 14px;
+			--action-button-padding: 2px 0 0 0;
+			--checked-color: var(--checked-color);
+		}
+		gl-onboarding-checkbox {
+			flex-shrink: 0;
+			margin-right: 8px;
+		}
+		gl-button {
+			--button-padding: var(--action-button-padding);
+		}
+		.actions code-icon {
+			font-size: 20px;
+			color: var(--action-button-color, gray);
+		}
+		.actions {
+			display: flex;
+			align-items: center;
+			margin: -4px 0;
+		}
+		.description {
+			display: inline-flex;
+			flex: 1;
+			align-items: center;
+		}
+		.description span {
+			flex: 1;
+		}
+		.description.disabled span {
+			color: var(--disabled-color, gray);
+		}
+		.actions .action {
+			width: 24px;
+			box-sizing: border-box;
+			display: flex;
+			align-items: center;
+			justify-content: center;
+		}
+	`;
+
+	@property({ type: Boolean })
+	checked = false;
+
+	@property({ type: String })
+	playHref = '';
+
+	@property({ type: String })
+	infoTitle = '';
+
+	@property({ type: String })
+	infoHref = '';
+
+	private get infoPresented() {
+		return Boolean(this.infoHref) || Boolean(this.infoTitle);
+	}
+
+	private renderPlay() {
+		return html`<a href=${this.playHref} class="action"
+			><gl-button appearance="toolbar" ><code-icon icon="play-circle"></gl-button></a>`;
```

Comment:
gl-button has an href attr

```suggestion
		return html`<gl-button href=${this.playHref} appearance="toolbar" ><code-icon icon="play-circle"></gl-button>`;
```

Reply:
used